model-server/src/test/kotlin/org/modelix/model/server/handlers/ModelReplicationServerTest.kt

```diff
@@ -365,4 +383,53 @@ class ModelReplicationServerTest {
             )
         }
     }
+
+    @Test
+    fun `getRepositoryBranch responds with version delta v2 if requested`() {
+        val repositoryId = RepositoryId("repo1")
+
+        runWithTestModelServer { _, fixture ->
+            fixture.repositoriesManager.createRepository(repositoryId, null)
+
+            val response = client.get {
+                url {
+                    appendPathSegments("v2", "repositories", repositoryId.id, "branches", "master")
+                }
+                accept(VersionDeltaStreamV2.CONTENT_TYPE)
+            }
+
+            response shouldHaveStatus HttpStatusCodes.STATUS_CODE_OK
+            response.shouldHaveContentType(VersionDeltaStreamV2.CONTENT_TYPE)
```

Comment:
Really consistent, that `shouldHaveContentType` is not marked as `infix` like `shouldHaveStatus` 😆

Reply:
I was already thinking about opening an upstream bug report for that missing syntax